### ISSUE TITLE
Error when adding to Ionic 5 project

### ISSUE THREAD

**Issue by Maud-VanDorssen-D2G-DEV** [ImpacGroup/cordova-plugin-zoomimageview#1]
It's great you've created an up-to-date version of an image viewer plugin for Cordova.

I have tried adding this plugin to my Ionic 5 application but I keep getting errors like these : 
```
[cordova] e: Incompatible classes were found in dependencies. Remove them from the classpath or use '-Xskip-metadata-version-check' to suppress errors
[cordova] e: /Users/maudvd/.gradle/caches/transforms-2/files-2.1/0607a133ebed385262c1e535e01a3690/jetified-kotlin-stdlib-common-1.6.0-RC.jar!/META-INF/kotlin-stdlib-common.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
[cordova] e: /Users/maudvd/.gradle/caches/transforms-2/files-2.1/2e79364d25f2149bc7dfbafe9903dbe5/jetified-kotlin-stdlib-1.6.0-RC.jar!/META-INF/kotlin-stdlib.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
[cordova] e: /Users/maudvd/.gradle/caches/transforms-2/files-2.1/598ef7cd263c38322add3a6cc4646fbb/jetified-core-ktx-1.6.0-api.jar!/META-INF/core-ktx_release.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.5.1, expected version is 1.1.15.
[cordova] e: /Users/maudvd/.gradle/caches/transforms-2/files-2.1/792bdb42138217dd90b233994b65ff19/jetified-kotlin-stdlib-jdk7-1.6.0-RC.jar!/META-INF/kotlin-stdlib-jdk7.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
[cordova] e: /Users/maudvd/.gradle/caches/transforms-2/files-2.1/7effc3922c7d11639802be346760d6cc/jetified-TouchImageView-3.1.1-api.jar!/META-INF/touchview_release.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.5.1, expected version is 1.1.15.
[cordova] e: /Users/maudvd/Documents/winkort/platforms/android/app/src/main/kotlin/de/impacgroup/zoomimageview/module/ZoomImageActivity.kt: (33, 13): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
[cordova] The class is loaded from /Users/maudvd/.gradle/caches/transforms-2/files-2.1/2e79364d25f2149bc7dfbafe9903dbe5/jetified-kotlin-stdlib-1.6.0-RC.jar!/kotlin/Unit.class
[cordova] e: /Users/maudvd/Documents/winkort/platforms/android/app/src/main/kotlin/de/impacgroup/zoomimageview/module/ZoomImageActivity.kt: (43, 93): Unresolved reference: java
[cordova] e: /Users/maudvd/Documents/winkort/platforms/android/app/src/main/kotlin/de/impacgroup/zoomimageview/module/ZoomImageActivity.kt: (47, 22): Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.6.0, expected version is 1.1.15.
```

I have added the plugin with this command : 
`ionic cordova plugin add cordova-plugin-zoomimageview`

And I have added these lines to my config.xml file : 
```
<preference name="GradlePluginKotlinEnabled" value="true" />
<preference name="GradlePluginKotlinCodeStyle" value="official" />
<preference name="GradlePluginKotlinVersion" value="1.6.0-RC" />
```

Is there anything I can do to use this plugin in an Ionic 5 + Cordova project ?

**Comment by ImpacGroup** [comment]
Hi,

do you have any other cordova plugins implemented that use Kotlin? 
Can you send use more information about your cordova setup? Which version of cordova and Cordova Android are you using?

**Comment by Maud-VanDorssen-D2G-DEV** [comment]
Here is my `ionic info` output : 
```
Ionic:

   Ionic CLI                     : 6.18.1 (/usr/local/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.9.1
   @angular-devkit/build-angular : 12.0.5
   @angular-devkit/schematics    : 12.2.13
   @angular/cli                  : 12.0.5
   @ionic/angular-toolkit        : 4.0.0

Cordova:

   Cordova CLI       : 10.0.0
   Cordova Platforms : android 9.1.0, ios 6.2.0
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 4.2.1, (and 17 other plugins)

Utility:

   cordova-res                          : 0.15.3
   native-run (update available: 1.5.0) : 1.4.1

System:

   ios-sim : 8.0.2
   NodeJS  : v12.18.2 (/usr/local/bin/node)
   npm     : 7.20.0
   OS      : macOS Big Sur
   Xcode   : Xcode 13.0 Build version 13A233
```

I don't know if my other plugins use Kotlin. But I have never had to add lines like these to my `config.xml` before : 
```
<preference name="GradlePluginKotlinEnabled" value="true" />
<preference name="GradlePluginKotlinCodeStyle" value="official" />
<preference name="GradlePluginKotlinVersion" value="1.6.0-RC" />
```

These are the cordova plugins I have installed : 
```
cordova-plugin-actionsheet 2.3.3 "ActionSheet"
cordova-plugin-advanced-http 3.2.2 "Advanced HTTP plugin"
cordova-plugin-androidx-adapter 1.1.3 "cordova-plugin-androidx-adapter"
cordova-plugin-camera 5.0.3 "Camera"
cordova-plugin-datepicker 0.9.3 "DatePicker"
cordova-plugin-device 2.0.2 "Device"
cordova-plugin-dialogs 2.0.2 "Notification"
cordova-plugin-file-transfer 1.7.1 "File Transfer"
cordova-plugin-file 6.0.2 "File"
cordova-plugin-filepath 1.6.0 "cordova-plugin-filepath"
cordova-plugin-firebasex 12.1.0 "Google Firebase Plugin"
cordova-plugin-geolocation 4.1.0 "Geolocation"
cordova-plugin-inappbrowser 5.0.0 "InAppBrowser"
cordova-plugin-ionic-keyboard 2.2.0 "cordova-plugin-ionic-keyboard"
cordova-plugin-ionic-webview 4.2.1 "cordova-plugin-ionic-webview"
cordova-plugin-purchase 10.6.1 "Purchase"
cordova-plugin-request-location-accuracy 2.3.0 "Request Location Accuracy"
cordova-plugin-safariviewcontroller 2.0.0 "SafariViewController"
cordova-plugin-splashscreen 6.0.0 "Splashscreen"
cordova-plugin-statusbar 2.4.3 "StatusBar"
cordova-plugin-whitelist 1.3.5 "Whitelist"
cordova.plugins.diagnostic 6.1.0 "Diagnostic"
uk.co.workingedge.phonegap.plugin.launchnavigator 5.0.6 "Launch Navigator"
```

**Comment by ImpacGroup** [comment]
As you can see in the documentation the plugins supports cordova-android 10 or above. You are using android 9.1.0 which is not supported. Please upgrade to cordova-android 10.
To check how the plugin is working we've added a small demo project.
